Description: added testing.sh
 .
 hello (2.10-3ubuntu1) UNRELEASED; urgency=medium
 .
   * Added testing.sh which is executed by postinst
Author: Andrea Gangemi <andrea@satrapi>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2025-07-05

--- /dev/null
+++ hello-2.10/testing.sh
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "this is a test from Andrea Gangemi" >&2
